cmd/embedded-cluster/flags.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/defaults"
+	"github.com/urfave/cli/v2"
+	k8snet "k8s.io/utils/net"
+)
+
+func getAdminColsolePortFlag() cli.Flag {
+	return &cli.StringFlag{
+		Name:   "admin-console-port",
+		Usage:  "Port on which the Admin Console will be served",
+		Value:  strconv.Itoa(defaults.AdminConsolePort),
+		Hidden: false,
+	}
+}
+
+func getAdminConsolePortFromFlag(c *cli.Context) (int, error) {
+	portStr := c.String("admin-console-port")
+	if portStr == "" {
+		return defaults.AdminConsolePort, nil
+	}
+	// TODO: add first class support for service node port range and validate the port
+	port, err := k8snet.ParsePort(portStr, false)
+	if err != nil {
+		return 0, fmt.Errorf("invalid admin console port: %w", err)
+	}
+	return port, nil
+}
+
+func getLocalArtifactMirrorPortFlag() cli.Flag {
+	return &cli.StringFlag{
+		Name:   "local-artifact-mirror-port",
+		Usage:  "Port on which the Local Artifact Mirror will be served",
```

Comment:
I was considering whether it was worth mentioning the name LAM or not, because an end user will have no idea what that is and doesn't need to. But I don't know what else we'd call it other than `port-2`. So maybe it's fine.

I don't have a strong preference, but we could consider `lam-port` if we feel like `local-artifact-mirror-port` is too long, or use it as an alias. People don't type this much, so I don't care much.